spec/abstractops.html

```diff
@@ -266,7 +240,9 @@ <h1>ToTemporalRoundingIncrement ( _normalizedOptions_, _dividend_, _inclusive_ )
         1. Let _maximum_ be _dividend_ − 1.
       1. Else,
         1. Let _maximum_ be 1.
-      1. Let _increment_ be ? GetNumberOption(_normalizedOptions_, *"roundingIncrement"*, 1, _maximum_, 1).
+      1. Let _increment_ be ? GetOption(_normalizedOptions_, *"roundingIncrement"*, « Number », *undefined*, 1).
+      1. If _increment_ &lt; 1 or _increment_ &gt; _maximum_, throw a *RangeError* exception.
```

Comment:
this also lacks an assertion that `dividend` is a Number (thus ensuring `maximum` is a Number), but also, `1` is a mathemetical value.

In other words, a series of assertions/conversions is required here so the number types are the same.

Reply:
I'd rather do this as part of #1413, to keep the scope of this change from expanding more than it already has...